angular-cli.json

```diff
@@ -18,9 +18,12 @@
       "tsconfig": "tsconfig.json",
       "prefix": "app",
       "styles": [
-        "styles.scss"
+        "styles.scss",
+        "../node_modules/bootstrap/dist/css/bootstrap.min.css",
+        "../node_modules/owl.carousel/dist/assets/owl.carousel.css",
+        "../node_modules/font-awesome/css/font-awesome.css"
       ],
-      "scripts": [],
+      "scripts": ["../node_modules/jquery/dist/jquery.js"],
```

Comment:
What for we need jQuery?

Reply:
it's necessary for owl carousel

Reply:
We can use pure/vanilla JS, for example:

> document.getElementById('owl-id-selector').owlCarousel({...HERE_SOME_OPTIONS...});